cmd/nerdctl/system_prune.go

```diff
@@ -99,5 +102,68 @@ func systemPruneAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	return imagePrune(cmd, client, ctx)
+	if err := imagePrune(cmd, client, ctx); err != nil {
+		return nil
+	}
+	prunedObjects, err := buildCachePrune(ctx, cmd, all)
+	if err != nil {
+		return nil
+	}
+
+	if len(prunedObjects) > 0 {
+		fmt.Fprintln(cmd.OutOrStdout(), "Deleted build cache objects:")
+		for _, item := range prunedObjects {
+			fmt.Fprintln(cmd.OutOrStdout(), item.ID)
+		}
+	}
+
+	// TODO: print total reclaimed space
+
+	return nil
+}
+
+type cacheUsageInfo struct {
+	ID   string
+	Size int64
+}
+
+func buildCachePrune(ctx context.Context, cmd *cobra.Command, pruneAll bool) ([]cacheUsageInfo, error) {
+	ns, err := cmd.Flags().GetString("namespace")
+	if err != nil {
+		return nil, err
+	}
+	buildkitHost, err := buildkitutil.GetBuildkitHost(ns)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should be aware of `BUILDKIT_HOST` (or you can use `getBuildkitHost` in build.go).

Reply:
Updated! PTAL